HomeWork_03/DZ3.py

```diff
@@ -1,35 +1,32 @@
 #I love Python*42#
 
 # Zadanie 1
-x = ("I love Python")
+x = "I love Python"
 z = x*42
 print(z)
 
 #age_in_month#
 
 # Zadanie 2
-x = str("age_in_month")
-y = 37*12
-z = x+" "+str(y)
+x = month = 12
+y = age_in_year = 37
+z = age_in_month = x*y
 print(z)
 
 #Take age in "y" Zadanie 2#
 
 # Zadanie 3
-i = str(" age_in_years")
-j = str(i)+" " + str(y//12)
+i = " age_in_years"
+j = i + " " + str(z//12)
 print (j)
 
 #----------------------#
 # Zadanie 4
 first_name = "Oleksandr"
 last_name = "Pitelin"
 full_name = first_name +" "+ last_name
-k = ("Му name is" +" "+ (full_name))
-l = ("I’m" + (j) + "years old")
-my_age =  k+" "+l
-
-print(my_age)
+result = "Му name is" +" "+ (full_name) + " I’m" + (j) + "years old"
```

Comment:
There is no need to use parentheses for `full_name`and you should use `age_in_years` instead of `j` here which is transformed to string: `str(age_in_years)`